sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/data/cosmos/CosmosReactiveRepositoriesAutoConfiguration.java

```diff
@@ -21,7 +21,7 @@
 @ConditionalOnClass({ ReactiveCosmosRepository.class })
 @ConditionalOnMissingBean({ ReactiveCosmosRepositoryFactoryBean.class,
     ReactiveCosmosRepositoryConfigurationExtension.class })
-@ConditionalOnBean(ReactiveCosmosTemplate.class)
+@AutoConfigureAfter(ReactiveCosmosTemplate.class)
```

Comment:
Is this change intentional? Is ReactiveCosmosTemplate an auto-configuration class?